third_party/2and3/markdown/blockprocessors.pyi

```diff
@@ -0,0 +1,78 @@
+from typing import Any
+
+logger: Any
+
+def build_block_parser(md, **kwargs): ...
+
+class BlockProcessor:
+    parser: Any
+    tab_length: Any
+    def __init__(self, parser) -> None: ...
+    def lastChild(self, parent): ...
+    def detab(self, text): ...
+    def looseDetab(self, text, level: int = ...): ...
+    def test(self, parent, block) -> None: ...
+    def run(self, parent, blocks) -> None: ...
+
+class ListIndentProcessor(BlockProcessor):
+    ITEM_TYPES: Any
+    LIST_TYPES: Any
+    INDENT_RE: Any
+    def __init__(self, *args) -> None: ...
+    def test(self, parent, block): ...
+    def run(self, parent, blocks) -> None: ...
+    def create_item(self, parent, block) -> None: ...
+    def get_level(self, parent, block): ...
+
+class CodeBlockProcessor(BlockProcessor):
+    def test(self, parent, block): ...
+    def run(self, parent, blocks) -> None: ...
+
+class BlockQuoteProcessor(BlockProcessor):
+    RE: Any
```

Comment:
I assume this is a `Pattern`?

Reply:
Made the change. Not all of them are Pattern, see https://github.com/python/typeshed/pull/4574/commits/8f2c0c556a5111551fc1df7649f36d2dcbcf6351